blog/2023-06-28-service-description-filters.md

```diff
@@ -0,0 +1,11 @@
+---
+author: Erik Montnémery
+authorURL: https://github.com/emontnemery
+title: "Service call description filters"
+---
+
+Service call fields now support filtering, to avoid showing service call parameters which are not supported by an entity to the user. As an example, the `brightness` service call parameter for `light.turn_on` will only be shown if the light supports brightness.
```

Comment:
```suggestion
Service call fields now support filtering, to avoid showing service call parameters that are not supported by an entity to the user. For example, the `brightness` service call parameter for `light.turn_on` will only be shown if the light supports brightness.
```